core/templates/dev/head/profile/Signup.js

```diff
@@ -67,7 +75,7 @@ oppia.controller('Signup', [
         username: $scope.username
       }).success(function(data) {
         if (data.username_is_taken) {
-          $scope.warningText = 'Sorry, this username is already taken.'
+          $scope.warningText = 'I18N_SIGNUP_ERROR_USERNAME_TAKEN'
```

Comment:
I feel a bit uncomfortable returning something that's essentially a variable here (although it looks like a string).

Do you think it would be OK to store this in "var warningType" instead, and, at the end, do something like $scope.warningText = $filter('translate')(warningType)?

Also /cc @BenHenning for second opinion (with an eye towards maintainability).